lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Možná okomentovat nápadnou podobnost mezi JSON Pythonem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Jo, s tím mívají lidi problém. Jakmile vidí složenou závorku, řeknou si "ha, slovník", a nějak to nedokáží zkombinovat s faktem, že to je jen řetězec nebo proč se to musí ještě nějak načítat. A naopak, proč převádět Python slovník do JSONu, když už to "skoro tak vypadá rovnou"?</p><p> </p><p>Formálně vzato - JSON není podmnožina Python syntaxe, dalo by se upozornit např. na to, že v něm může být "true", "false", "null", a to v Pythonu není.</p>